include/codec/util.hpp

```diff
@@ -457,39 +480,6 @@ namespace SystemUtils {
 
 namespace FileUtils {
 
-/**
```

Comment:
This hints to our need to cleanup the utils header. Much of this is Task specific